### ISSUE TITLE
label indices mismatch between evaluation and gt data 

### ISSUE THREAD

**Issue by olive0206** [hellochick/ICNet-tensorflow#10]
Hello~,

I recently downloaded the cityscape dataset from the official website.

and I tried to run evaluation.py code with cityscape eval & gt data.

Fortunately, I produce the the prediction result from the evaluation code using your trained model provided on GoogleDrive.

But Badly, I found the label indices generated from evaluation and those from ground-truth are mismatched. For example, the index for class person is 11 in your dataset, but it is 24 in cityscape dataset. so the final mIOU is extremely low like 0.00XX...

I guess, you trained the model with old version of cityscape dataset or something different from mine.

Am I right?? could you confirm my opinion??

**Comment by hellochick** [comment]
Hey, @olive0206 , since I have converted the dataset into trainID format. You can take a look at following repo, especially this code: https://github.com/mcordts/cityscapesScripts/blob/master/cityscapesscripts/helpers/labels.py
It define the mapping between classes !

**Comment by olive0206** [comment]
Thanks a lot, @hellochick !! ;)
